framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -189,6 +193,53 @@ private AnnotationMirror createBooleanAnnotationMirror(List<Boolean> values) {
         return new RegularTransferResult<>(newResultValue, result.getRegularStore());
     }
 
+    private TransferResult<CFValue, CFStore> createNewConditionalResult(
```

Comment:
Is a new method really needed or could `createNewResultBoolean` be changed to return a conditional result (when appropriate)?